spyder_terminal/terminalplugin.py

```diff
@@ -47,141 +30,74 @@
 _ = get_translation('spyder_terminal')
 
 
-class TerminalPlugin(SpyderPluginWidget):
-    """Terminal plugin."""
-
-    URL_ISSUES = ' https://github.com/spyder-ide/spyder-terminal/issues'
-    CONF_SECTION = CONF_SECTION
-    CONFIGWIDGET_CLASS = TerminalConfigPage
+class TerminalPlugin(SpyderDockablePlugin):
+    """
+    Terminal plugin.
+    """
+    NAME = 'terminal'
+    REQUIRES = [Plugins.WorkingDirectory, Plugins.Preferences]
```

Comment:
According to what I can see in the `register` method below, `WorkingDirectory` should be optional and not required.